api/src/database/channel-data.ts

```diff
@@ -0,0 +1,38 @@
+import { CollectionNames } from './constants';
+import { MongoDbService } from '../services/mongodb-service';
+import { ChannelData } from '../models/types/channel-data';
+
+const collectionName = CollectionNames.channelData;
+const getIndex = (link: string, userId: string) => `${link}-${userId}`;
+
+export const getChannelData = async (channelAddress: string, userId: string, limit?: number, index?: number): Promise<any> => {
```

Comment:
Could you also define which value type do you expect to get back instead of `Promise<any>`?